mkdocs.yml

```diff
@@ -26,16 +26,17 @@ theme:
 
 nav:
   - Home: 'index.md'
-  - Quickstart: 
-    - Quickstart: 'quickstart/quickstart.md'
-    - Initialization: 'quickstart/init.md'
-    - Build and Setup: 'quickstart/build_setup.md'
-    - Run: 'quickstart/server.md'
-    - Configuration: 'quickstart/config.md'
-    - Customization: 'quickstart/customize.md'
-    - Troubleshooting: 'quickstart/troubleshoot.md'
-    - Cleanup: 'quickstart/cleanup.md'
-  - Deployment:
+  - Install: 'install/index.md'
+  - Preview: 'preview/index.md'
+  - Develop:
+    - Getting Started: 'develop/index.md'
+    - Install Locally: 'develop/install_locally.md'
+    - Run: 'develop/run.md'
+    - Configuration: 'develop/config.md'
+    - Customization: 'develop/customize.md'
+    - Troubleshooting: 'develop/troubleshoot.md'
+    - Cleanup: 'develop/cleanup.md'
+  - Deploy:
     - Kubernetes: 'deployment/kubernetes.md'
```

Comment:
minor: It goes directly to Kubernetes. Maybe adding an `index.md` saying "Coming soon" would be better not to confuse. Once again, this is UX (Not your task hehe) so to save time if so just open an issue and we will revisit it later on.

Reply:
Added "Coming soon" to the Kubernetes and Openshift docs. I don't know yet how to use `index.md` correctly...